nncf/graph/graph.py

```diff
@@ -80,6 +81,10 @@ def __init__(self, node_id: int,
     @property
     def node_type(self):
         if self.ia_op_exec_context:
+            if self.ia_op_exec_context.operator_name == MODEL_INPUT_OP_NAME:
+                return NNCFGraphNodeType.INPUT_NODE
+            if self.ia_op_exec_context.operator_name == MODEL_OUTPUT_OP_NAME:
+                return NNCFGraphNodeType.OUTPUT_NODE
             return self.ia_op_exec_context.operator_name
         return None
```

Comment:
A node_type is currently a `str`. You introduced an integer-valued enum as an extra possible type. The return types should be consistent.

Reply:
Done.